src/__test__/operate.test.js

```diff
@@ -0,0 +1,15 @@
+import calculate from '../logic/calculate';
+
+it('checks calculate', () => {
+  const object1 = {
+    total: null,
+    next: null,
+    operation: null,
+  };
+
+  expect(calculate(object1, 'AC')).toEqual({
+    total: null,
+    next: null,
+    operation: null,
+  });
+});
```

Comment:
- [ ] This is exactly the same test we have in `calculate.test.js`. It, therefore, doesn't count as a test since it is a repetition. That one said we are supposed to import the `operate.js` into this test file and test all the operations in it which include `+`, `-`, `x`, `÷`, and `%`. Please write tests for these functions.